src/core/server/elasticsearch/elasticsearch_client_config.test.ts

```diff
@@ -87,9 +83,9 @@ test('parses fully specified config', () => {
     sniffInterval: 11223344,
     ssl: {
       verificationMode: 'certificate',
-      certificateAuthorities: ['ca-path-1', 'ca-path-2'],
-      certificate: 'certificate-path',
-      key: 'key-path',
+      certificateAuthorities: ['content-of-ca-path-1', 'content-of-ca-path-2'],
+      certificate: 'content-of-certificate-path',
+      key: 'content-of-key-path',
```

Comment:
`elasticsearch_client_config` no longer reads the contents of these files -- `elasticsearch_config` handles that. So we don't need to mock the `readFileSync` implementation here and we can assume that the contents of these certs/keys are being passed in to the `ElasticsearchClientConfig` params.